code/style.css

```diff
@@ -1,4 +1,334 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* Header */
+
+h1 {
+  margin: 0;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+nav {
+  background-color: white;
+  color: black;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-bottom: solid;
+  border-color: cornflowerblue;
+}
+
+.leftlinks {
+  display: none;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.rightlinks {
+  display: none;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+.rightlinks:hover {
+  color: blue;
+}
+
+.leftlinks:hover {
+  color: blue;
+}
+
+.logo {
+  height: 50px;
+  margin: 10px 20px;
+}
+nav {
+  display: flex;
+}
```

Comment:
Keep all the nav styling in one CSS declaration (you already have one `nav` styling section)